### DISCUSSION TITLE
Niko's availability over the rest of April

### DISCUSSION THREAD

**Discussion by nikomatsakis** [lalrpop/lalrpop#738] [General]
Hey @lalrpop/maintainers I just wanted to mention that I've been pretty busy at work the last week and that I'm going to be unavailable until around May for various reasons. I'll try to keep up with GH notifications (though I've let those slide) but I don't expect to do much more.

**Comment by yannham** [comment]
Hey, while we're at it, I will be mostly unavailable in May, on my side.
